docs/install.rst

```diff
@@ -23,6 +23,10 @@ This is to override the ``jquery.init.js`` script provided by the admin, which
 sets up jQuery with ``noConflict``, making jQuery available in
 ``django.jQuery`` only and not ``$``.
 
+Note: If you are trying to install from git, please make sure you are not using **zip/archive** url of \ 
+the repo ``django-autocomplete-light`` since it will not contain required submodules automatically. \
```

Comment:
Could you remove the trailing `\` and word wrap to 80 caracters ? (if using vim, select the paragraph in visual mode and type gqq)